core/src/main/scala/filodb.core/downsample/DownsampledTimeSeriesShard.scala

```diff
@@ -192,6 +192,8 @@ class DownsampledTimeSeriesShard(rawDatasetRef: DatasetRef,
     // records with same part key. This is true since we update part keys only once per flush interval in raw dataset.
     logger.info(s"Starting housekeeping for downsample cluster of dataset=$rawDatasetRef shard=$shardNum " +
       s"every ${rawStoreConfig.flushInterval}")
+    require(downsampleStoreConfig.readerRefreshInterval.isEmpty,
```

Comment:
Why put this assertion? Currently refresh in downsample shard is performed as part of housekeeping task which purges expired data.

Reply:
That's right-- I'm just being explicit here to prevent someone in the future from assuming "readerRefreshInterval" will affect the downsample store.